website/client/src/components/challenges/sidebar.vue

```diff
@@ -34,7 +34,7 @@
           </div>
         </div>
       </filter-group>
-      <filter-group :title="$t('role')">
+      <!-- <filter-group :title="$t('role')">
```

Comment:
OK to delete these rather than commenting out. Git history will preserve it if we need to restore it!